localization/english/activities/expeditions/spark_trial_of_cunning_l_english.yml

```diff
@@ -1,8 +1,8 @@
 ﻿l_english:
   # Trial of cunning
-  expedition.00512.title:0 "Trial of Cunning"
-  expedition.00512.desc:0 "You enter a grand chamber. Across the room from you stands a statue of fine white marble depicting an Elf Maiden. Between you the floor looks to be some type of game board. You notice carvings, depicting a variety of figures. Seems you need to play a game of some sort to pass this trial.\n\nAs you take a step forward you are surprised when the maiden begins to move. She gives you a pretty smile that conveys a clear message. #warning Your Move#!  \n\n"
-  
+  expedition.00512.title:0 "A Trial of Cunning"
+  expedition.00512.desc:0 "Upon entering a grand chamber, you spot a statue of fine white marble directly across from you. It depicts a fine Elf Maiden, overlooking what appears to be a game board. You notice some carvings of a variety of figures embedded into the tiles; it seems as though you need to play a game of some sort in order to pass this trial. \n\nAs you take a step forward, the maiden's sudden movement causes you to jump back. She flashes you a pretty smile that conveys a clear message;\n\n #warning Your Move#! \n\n"
```

Comment:
> It depicts a fine Elf Maiden

I would drop the word "fine" since its used in the previous sentence. I think "It depicts an Elf Maiden" is good unless you want to add a replacement adjective